TestStack.Dossier.Tests/TestHelpers/StaticAnonymousValueSupplier.cs

```diff
@@ -14,9 +16,19 @@ public StaticAnonymousValueSupplier(object valueToSupply)
             return typeof(TValue) == _valueToSupply.GetType();
         }
 
+        public bool CanSupplyValue(Type type, string propertyName)
```

Comment:
I've done all the changes apart from this. I agree it makes sense to change this, but it might be an idea to hold off on the breaking change and see if there might be more features we want to add for v3...?